app/controllers/react_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ReactController < ApplicationController
   layout 'layouts/react_application'
+
+  def index
```

Comment:
@ohadlevy 
I had to create an empty `index.html.erb` for our structure to work. if not it gives me the following error:

> ActionController::UnknownFormat in ProfilesController#index
ProfilesController#index is missing a template for this request format and variant. request.formats: ["text/html"] request.variant: [] NOTE! For XHR/Ajax or API requests, this action would normally respond with 204 No Content: an empty white screen. Since you're loading it in a web browser, we assume that you expected to actually render a template, not… nothing, so we're showing an error to be extra-clear. If you expect 204 No Content, carry on. That's what you'll get from an XHR or API request. Give it a shot.